presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/model/GroupKeyWord.kt

```diff
@@ -0,0 +1,48 @@
+package com.mashup.gabbangzip.sharedalbum.presentation.ui.model
+
+import androidx.compose.ui.graphics.Color
+import com.mashup.gabbangzip.sharedalbum.presentation.R
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.ConiferAlpha30
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.CoralAlpha30
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.DandelionAlpha30
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.LavenderAlpha30
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.MagentaPinkAlpha30
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.MalibuAlpha30
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.MayaBlueAlpha30
+
+enum class GroupKeyWord {
+    SCHOOL,
+    COMPANY,
+    CREW,
+    NETWORK,
+    EXERCISE,
+    HOBBY,
+    LITTLE_MOIM,
+    ;
+
+    companion object {
+        fun getSymbol(keyword: GroupKeyWord): Int {
+            return when (keyword) {
+                SCHOOL -> R.drawable.sb_school
+                COMPANY -> R.drawable.sb_company
+                CREW -> R.drawable.sb_crew
+                NETWORK -> R.drawable.sb_network
+                EXERCISE -> R.drawable.sb_exercise
+                HOBBY -> R.drawable.sb_hobby
+                LITTLE_MOIM -> R.drawable.sb_little_moim
+            }
+        }
+
+        fun getBackgroundColor(keyword: GroupKeyWord): Color {
+            return when (keyword) {
+                SCHOOL -> ConiferAlpha30
+                COMPANY -> MagentaPinkAlpha30
+                CREW -> MayaBlueAlpha30
+                NETWORK -> CoralAlpha30
+                EXERCISE -> DandelionAlpha30
+                HOBBY -> MalibuAlpha30
+                LITTLE_MOIM -> LavenderAlpha30
+            }
+        }
+    }
+}
```

Comment:
```suggestion
enum class GroupKeyWord(@DrawableRes val symbolResId: Int, val backgroundColor: Color){
    SCHOOL(R.drawable.sb_school, ConiferAlpha30),
    COMPANY(R.drawable.sb_company, MagentaPinkAlpha30),
    CREW(R.drawable.sb_crew, MayaBlueAlpha30),
    NETWORK(R.drawable.sb_network, CoralAlpha30),
    EXERCISE(R.drawable.sb_exercise, DandelionAlpha30),
    HOBBY(R.drawable.sb_hobby, MalibuAlpha30),
    LITTLE_MOIM(R.drawable.sb_little_moim, LavenderAlpha30),
    ;
}
```
함수 만드는 것 보다 enum class 정의하는 시점에 이렇게 정의하는게 더 깔끔할 것 같아요~!

Reply:
그으런데 이렇게 하면 타입별로 구분할 수가 있어?? 이렇게 되면 서버에서 타입을 내려줄텐데 사용하는 곳에서 계속 판별해줘야 되지 않아??